src/AzureClient/EntryPoint/EntryPointOperationResolver.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Quantum.IQSharp.AzureClient
+{
+    internal class EntryPointOperationResolver : IOperationResolver
+    {
+        private IEntryPointGenerator EntryPointGenerator { get; }
+
+        public EntryPointOperationResolver(IEntryPointGenerator entryPointGenerator) =>
+            EntryPointGenerator = entryPointGenerator;
+
+        public OperationInfo Resolve(string name) => OperationResolver.ResolveFromAssemblies(name, RelevantAssemblies());
+
+        private IEnumerable<AssemblyInfo> RelevantAssemblies()
```

Comment:
I'm assuming that relevant assemblies does not need to include the info for the entrypoint assembly itself, right?
(EntryPointAssemblyInfo)

Reply:
Correct.  Any operation that we want to resolve here must live in one of the other assemblies.